heron/tools/cli/src/python/args.py

```diff
@@ -41,18 +41,6 @@ def add_verbose(parser):
   return parser
 
 
-def add_trace_execution(parser):
-  '''
-  :param parser:
-  :return:
-  '''
-  parser.add_argument(
-      '--trace-execution',
```

Comment:
It is not used. Pretty sure it is some leftover when we move from close-sourced to open-sourced Heron.